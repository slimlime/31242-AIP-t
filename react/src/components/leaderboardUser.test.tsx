import React from "react";
import { shallow } from "enzyme";
import LeaderboardUser from "./leaderboardUser";

// simple, static component
describe("<LeaderboardUser />", () => {
  it("should render correctly", () => {
    const wrapper = shallow(
      <LeaderboardUser rank={1} username={"Kevin"} score={69} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
