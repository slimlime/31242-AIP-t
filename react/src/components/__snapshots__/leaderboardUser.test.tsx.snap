// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeaderboardUser /> should render correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  direction="row"
>
  <WithStyles(ForwardRef(Grid))
    id="leaderboardItem"
    item={true}
    xs={2}
  >
    <WithStyles(ForwardRef(Typography))>
      1
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    id="leaderboardItem"
    item={true}
    xs={5}
  >
    <WithStyles(ForwardRef(Typography))>
      Kevin
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    id="leaderboardItem"
    item={true}
    xs={2}
  >
    <WithStyles(ForwardRef(Typography))>
      69
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    id="leaderboardItem"
    item={true}
    xs={3}
  >
    <WithStyles(ForwardRef(Avatar))
      id="leaderboardUserAvatar"
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "center",
        }
      }
    >
      K
    </WithStyles(ForwardRef(Avatar))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
